integration/default_set.yaml

```diff
@@ -152,11 +152,13 @@ tests:
     steps:
       - command: import -b --table-slice-type=caf zeek
         input: data/zeek/conn.log.gz
-      - command: import -b --table-slice-type=arrow zeek
-        input: data/zeek/conn.log.gz
       - command: import -b --table-slice-type=msgpack zeek
         input: data/zeek/conn.log.gz
       - command: count '#type == "zeek.conn"'
+      - guard: 'export -h 2>&1 | grep -q arrow'
```

Comment:
Could this be reformulated more idiomatically by using the `version` command?

Reply:
Yeah, like `version | jq -re '."Apache Arrow" != null'

Reply:
I've implemented this change.

Reply:
The push somehow doesn't show up in here, I think the GitHub UI is broken.